hadoop/README.md

```diff
@@ -0,0 +1,36 @@
+# Spring Boot Trading REST API
+* [Introduction](#Introduction)
+* [Hadoop Cluster](#HadoopCluster)
+* [Hive Project](#HiveProject)
+* [Improvements](#Improvements)
+
+# Introduction
+ This project processes data using Apache Hadoop and evaluate different tools. It helped me understanding the core concepts of HDFS, MapReduce and hive. Google CLoud Platform is used build hadoop cluster: master node and slave node. YARN manages both the nodes. User is able to connect with hive server by CLI via beeline or by Zeppelin Notebook. Data within google storage is first written to HDFS and is manupilated using HiveQL.
+ 
+ # Hadoop Cluster
+![Architecture](/hadoop/assets/architecture.png)
```

Comment:
some connections are missing for hive components

Reply:
![image](https://user-images.githubusercontent.com/58951265/76858860-a0931700-682e-11ea-9982-7eba5a0b7b76.png)